dummy-bindings.rs

```diff
@@ -0,0 +1,3677 @@
+/* automatically generated by rust-bindgen */
```

Comment:
Please document what version of julia.h was used to generate these bindings in the module documentation of lib.rs, I think it's something that needs to be explicit to prevent confusion.

Reply:
It turns out to not be possible to embed the `//!` comment in the bindings file itself, but I have added a comment to the main `lib.rs` file:
```
//! The documentation found on docs.rs corresponds to Julia version 1.3.1, however when
//! compiled locally, the bindings will match the version installed locally.
```

Reply:
That's what I was asking for :) That keeps this file completely generated